user-org-util/src/main/java/org/sunbird/util/responsecode/ResponseCode.java

```diff
@@ -0,0 +1,39 @@
+package org.sunbird.util.responsecode;
+
+public enum ResponseCode {
+  internalError("INTERNAL_ERROR", "Process failed,please try again later.");
```

Comment:
Put in Error code and Error response message enum.
"INTERNAL_ERROR", "Process failed,please try again later."